fix for compiler errors when using VTK v9.0.1 as dependency:
    common/shape.cpp:359:61: error: binding reference of type ‘const vtkIdType*&’ {aka ‘const long long int*&’} to ‘vtkIdType*’ {aka ‘long long int*’} discards qualifiers
author: Kenneth Hoste (HPC-UGent)
--- fsl/src/mist/common/shape.cpp.orig	2022-06-24 19:10:57.716622000 +0200
+++ fsl/src/mist/common/shape.cpp	2022-06-24 19:11:04.593519590 +0200
@@ -354,7 +354,7 @@
     for (vtkIdType i = 0; i < cells->GetNumberOfIds(); i++)
     {
         vtkIdType npoints;
-        vtkIdType *points;
+        const vtkIdType *points;
 
         m_polyData->GetCellPoints(cells->GetId(i), npoints, points);
 
